frontend/src/components/EditRecipe/EditRecipeForm.js

```diff
@@ -11,16 +12,38 @@ const EditRecipeForm = ({ history }) => {
   const [difficulty, setDifficulty] = useState('ADVANCED');
   const [duration, setDuration] = useState('30 mins');
   const [servings, setServings] = useState('2');
+  const [ingredients, setIngredients] = useState(['']);
 
   const handleSubmit = () => {
     const parsedDuration = parseInt(duration, 10);
     const parsedServings = parseInt(servings, 10);
 
     createRecipe({
-      title, description, difficulty, duration: parsedDuration, servings: parsedServings,
+      title,
+      description,
+      difficulty,
+      duration: parsedDuration,
+      servings: parsedServings,
+      ingredients,
     }).then(response => history.push(`/recipe/${response.data.id}`));
   };
 
+  const handleChange = (i, event) => {
+    const values = [...ingredients];
+    values[i] = event.target.value;
+    setIngredients(values);
+  };
+  const handleAdd = () => {
+    const values = [...ingredients];
+    values.push('');
+    setIngredients(values);
+  };
+  const handleRemove = (i) => {
```

Comment:
I'd probably change the names of all these functions to be `handleIngredientChange`, etc. We'll probably need other `handleChange` functions on this page.

Reply:
What would you think about moving these functions and corresponding lines of JSX into their own separate file? I was getting started on the recipe description ticket and am realizing how unwieldy this EditRecipeForm is becoming.

Reply:
@vhoof